addon/model.js

```diff
@@ -50,7 +50,7 @@ var Model = Ember.Object.extend({
         this._super();
         this._reset();
         this._setup();
-        this.set("_data", {});
+        this.set("_data", clone(this));
```

Comment:
interesting find -sorry I didn't see this last night when I was looking at the other bug w/ rollback